src/pages/components/lead-space-search.mdx

```diff
@@ -13,129 +13,161 @@ import ComponentFeedback from 'components/ComponentFeedback';
 
 <AnchorLinks>
 
-<AnchorLink>Resources</AnchorLink>
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Options</AnchorLink>
 <AnchorLink>Anatomy</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
 <AnchorLink>Behaviors</AnchorLink>
+<AnchorLink>Resources</AnchorLink>
 <AnchorLink>Content guidance</AnchorLink>
+<AnchorLink>Related components</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
-<ResourceLinks name="Lead space search" type="layout" />
+<InlineNotification>
+
+**Note:** In v2, lead space search no longer supports a media background to ensure the space remains accessible across breakpoints. The adjacent theme modifier has also been simplified for implementation.
+
+</InlineNotification>
 
 ## Overview
 
-Lead space search includes a heading, a typeahead search field, an optional subheading, an optional paragraph, and an optional background image.
+Lead space search includes a heading, an optional subheading, and a Carbon [search](https://carbondesignsystem.com/components/search/usage/) field. This version of the lead space is especially useful when the page displays many options that a user could be interested in.
+
+<Row>
 
-Typeahead is a language prediction tool that provides suggestions for users as they type in the search field. It is also known as autocomplete, incremental search, or search-as-you-type.
+<Column colMd={6} colLg={8}>
 
-![A visual example of the Lead space search component](../../images/component/lead-space-search/lead-space-search-default.jpg)
+![A visual example of the Lead space search component](../../images/component/lead-space-search/lead-space-search-overview.png)
 
 <Caption>Example of the lead space search component</Caption>
 
-### Use case
-
-![A visual example of typeahead results being shown by the typehead search from within the lead space search compoonent](../../images/component/lead-space-search/lead-space-search-usecase.jpg)
+</Column>
 
-<Caption>
-  Example of lead space search, showing the typeahead feature. Typeahead
-  suggests results as the user types in the search field.
-</Caption>
+</Row>
 
 ### When to use
 
-The search box is one of the most frequently used design elements in a content-heavy web experience. To be effective, the search should be displayed prominently as it can be the fastest route to information discovery. Finding information quickly is essential for reducing bounce, extending the time users spend on the page, and can also motivate them to take action.
+The search field is one of the most frequently used design elements in a content-heavy web experience. To be effective, the search should be displayed at the top of the page content as it can be the fastest route to information discovery. Finding information quickly is essential for reducing bounce rate, extending the time users spend on the page, and can motivating them to take action.
```

Comment:
```suggestion
The search field is one of the most frequently used design elements in a content-heavy web experience. To be effective, the search should be displayed at the top of the page content as it can be the fastest route to information discovery. Finding information quickly is essential for reducing bounce rate, extending the time users spend on the page, and can motivate them to take action.
```